test/Evaluate/CMakeLists.txt

```diff
@@ -17,6 +17,7 @@ add_executable(leading-zero-bit-count-test
 
 target_link_libraries(leading-zero-bit-count-test
   FortranEvaluateTesting
+  LLVMSupport
```

Comment:
I've added LLVMSupport in all the target_link_libraries that used FortranEvaluateTesting, because FortranEvaluateTesting used testing.cpp that used llvm::outs() and llvm::errs().